man/docker-images.1.md

```diff
@@ -61,6 +61,11 @@ The list will contain the image repository name, a tag for the image, and an
 image ID, when it was created and its virtual size. Columns: REPOSITORY, TAG,
 IMAGE ID, CREATED, and VIRTUAL SIZE.
 
+The ``docker images`` command takes an optional ``[REPOSITORY[:TAG]]``
+argument that will filter the list of images for the given repository.
+
```

Comment:
Oh, darn. Almost forgot the man page; can you update this description as well, based on the new text?